src/components/filteredSearch/InputFilter.tsx

```diff
@@ -0,0 +1,56 @@
+import styles from './InputFilter.module.scss';
+import { FC, useEffect, useRef, useState } from 'react';
+import Input from '@/components/UI/Input';
+import { NotParameteredTranslationKey, useAppTranslation } from '@/lib/i18n';
+import { BaseFilterProps } from '@/components/filteredSearch/FilteredSearch';
+
+export function InputFilter({
+  onUpdate,
+  forcedValue,
+  placeholder,
+  Icon,
```

Comment:
Pourquoi tu mets des majuscules à ces pptés là ? C'est parce qu'on les appelle en dessous dans une balise ? Dans ce cas là pour satisfaire les 2 conventions on peut faire comme ça nan ?
```suggestion
  icon: Icon,
```